deploy/deploy.go

```diff
@@ -440,6 +440,21 @@ func (lf *localFile) MD5() []byte {
 	return lf.md5
 }
 
+// knownHiddenDirectory checks if the specified name is a well known
+// hidden directory.
+func knownHiddenDirectory(name string) bool {
```

Comment:
I used this approach to open the door for configurable 'well known' directories in the future.